python/chronos/src/bigdl/chronos/data/tsdataset.py

```diff
@@ -473,8 +476,8 @@ def gen_rolling_feature(self,
         return self
 
     def roll(self,
-             lookback,
-             horizon,
+             lookback=None,
+             horizon=None,
```

Comment:
horizon should not be None at any circumstances.

Reply:
lookback can be set to 'auto' in default. Please also state clearly what 'auto' means in the API document.